requirements/test.txt

```diff
@@ -5,7 +5,7 @@ ddt==1.0.0
 factory_boy==2.8.1
 freezegun==0.3.9
 mock==2.0.0
-moto==1.0.1
+moto==0.4.31
```

Comment:
This is the latest version of `moto` that does not also include a dependency on `boto3`